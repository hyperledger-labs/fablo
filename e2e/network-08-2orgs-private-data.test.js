const { performTests } = require('./performTests');

const label = 'network-08-2orgs-private-data';

describe(label, () => {
  performTests(label, 'fabricaConfig-2orgs-private-data-2chaincodes.json');
});
