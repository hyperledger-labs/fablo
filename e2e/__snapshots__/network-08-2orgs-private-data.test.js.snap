// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`samples/fablo-config-hlf1.4-2orgs-2chaincodes-private-data.yaml should create proper files from samples/fablo-config-hlf1.4-2orgs-2chaincodes-private-data.yaml 1`] = `Array []`;
